### ISSUE TITLE
Release plan - v20.x Current

### ISSUE THREAD

**Issue by RafaelGSS** [nodejs/Release#855]
_Draft schedule - all dates subject to change_

Version | Release Date | Releaser
--------|--------------| ---------
v20.0.0 | 2023-04-18 |  @RafaelGSS
v20.1.0 | 2023-05-01 |

**Comment by RafaelGSS** [comment]
I can't "pin issue". Github limits the pinned issues by 3.

**Comment by TomasHubelbauer** [comment]
@targos Looks like 20.1.0 is slated for release today. Can we expect it will appear on https://nodejs.org/en/download/current today as well or does it take a few days to show up there? I am awaiting this release because it should fix a bug with ESM loaders that I am running into (#47566) hence my curiosity about the release timeline. Thank you!

**Comment by targos** [comment]
It's unlikely. I'm aiming for a release tomorrow, May 3.

**Comment by TomasHubelbauer** [comment]
Looks like 20.1.0 is up for download! Thank you

**Comment by targos** [comment]
I'm moving the release to Monday next week (05 June)

**Comment by gabrielschulhof** [comment]
I'm sorry if this is the wrong place to ask, but is it possible to include https://github.com/nodejs/node/pull/48933 in 20.6.0? It merges cleanly as of right now.

**Comment by RafaelGSS** [comment]
It should be included, yeah.

**Comment by RafaelGSS** [comment]
FWIW v20.6.0 might be delayed by one day (16-08-2023). @ulisesgascon is working on it.

**Comment by MrDee5556** [comment]
Comment

**Comment by MrDee5556** [comment]
[]()``

**Comment by MrDee5556** [comment]
> _Draft schedule - all dates subject to change_
> 
> Version | Release Date | Releaser
> --------|--------------| ---------
> v20.0.0 | 2023-04-18 |  @RafaelGSS
> v20.1.0 | 2023-05-03 | @targos
> v20.2.0 | 2023-05-16 | @targos
> v20.3.0 | 2023-06-05 | @targos / @RafaelGSS 
> v20.3.1 | 2023-06-20 | @RafaelGSS (security)
> v20.4.0 | 2023-07-04 | @RafaelGSS
> v20.5.0 | 2023-07-18 | @juanarbol 
> v20.5.1 | 2023-08-08 | @RafaelGSS (security)
> v20.6.0 | 2023-08-15 | @RafaelGSS 
> v20.7.0 | 2023-08-29 | 
> v20.8.0 | 2023-09-12 | 
> v20.9.0 | 2023-09-26 | 
> v20.10.0 | 2023-10-10 | 
> v20.11.0 | 2023-10-24 | 
> 
>

**Comment by Miguel736j** [comment]
His language will forever language will forever fix our problems

**Comment by UlisesGascon** [comment]
FWIW v20.6.0 is delayed until Monday (21-08-2023) because we need to run CITGM/CI/V8-CI.

**Comment by RafaelGSS** [comment]
@nodejs/releasers folks, we need volunteers for the next v20.x release. Unfortunately, I can't prepare any release until the end of the month. Since we've just released v20.6.0, I'm pushing the v20.7.0 release date by a week (there are a bunch of commits already).

**Comment by UlisesGascon** [comment]
In Oct I won't be available, but I am happy to take leadership or at least prepare v20.7.0

**Comment by targos** [comment]
Anyone for a release tomorrow or later this week?

**Comment by Tomengo18** [comment]
****

**Comment by ruyadorno** [comment]
I might be able to work on it Wed-Thu if no one else is available to pick it up 😊

**Comment by krk** [comment]
Will v20.9.0 be released before LTS?

**Comment by richardlau** [comment]
> Will v20.9.0 be released before LTS?

No, due to releaser availability we weren't able to a release before LTS. The LTS transition release will be Node.js 20.9.0.

**Comment by OrenXD** [comment]
To which resources this project release refer to?
so far I was able to see node 20 preview only on azure function, but actually we are really waiting for app-service support, where is it in roadmap?

**Comment by gabrielschulhof** [comment]
Can we please add https://github.com/nodejs/node/pull/50991to the next 20.x release?

**Comment by sosoba** [comment]
Also this fix is needed:
* [ watch: fix null fileName on windows systems #49891 ](https://github.com/nodejs/node/pull/49891)

**Comment by gabrielschulhof** [comment]
Could we please have https://github.com/nodejs/node/pull/51801 in the next release? It will help us with our next release of Node-API.

**Comment by richardlau** [comment]
It will depend on timing. https://github.com/nodejs/node/pull/51801 needs to go out in a current release first before being eligible for LTS, and usually be in a current release for two weeks. The next current release looks likely to be Node.js 22.0.0.

**Comment by marco-ippolito** [comment]
A release was scheduled for this month I can volunteer for next week

**Comment by marco-ippolito** [comment]
I can volunteer for 20.15.0 for the week around 20th of June, if anyone else wants to do it

**Comment by marco-ippolito** [comment]
If nobody else wants to I can volunteer for 10th~ of July

**Comment by marco-ippolito** [comment]
> If nobody else wants to I can volunteer for 10th~ of July

Since the security release has been delayed I guess I'll move v20.16.0 2 weeks after the security release right?

**Comment by targos** [comment]
I'll prepare a release for the 1st of October

**Comment by ranisalt** [comment]
I see that the README lists 2024-10-22 as the maintenance start date of v20, but it is still marked as LTS. So what is the current state of it? It's... unsettling to not have any LTS version at the moment.

**Comment by richardlau** [comment]
> I see that the README lists 2024-10-22 as the maintenance start date of v20, but it is still marked as LTS. So what is the current state of it? It's... unsettling to not have any LTS version at the moment.

I opened https://github.com/nodejs/Release/pull/1048 last night to update the status. 

Node.js 22 will become LTS on Tuesday (29 Oct) with the [22.11.0 release](https://github.com/nodejs/node/pull/55504) -- I'll merge https://github.com/nodejs/Release/pull/1049 after that release to update the README.

**Comment by targos** [comment]
>  It's... unsettling to not have any LTS version at the moment.

Maintenance is still LTS.

**Comment by marco-ippolito** [comment]
I can volunteer for v20.18.2 in january

**Comment by marco-ippolito** [comment]
I can volunteer for the next v20.x (the require esm)

**Comment by codebytere** [comment]
Hey folks - is there an estimated date for the next v20 release?

**Comment by marco-ippolito** [comment]
Unless someone wants to volunteer for it, I can do a release the first week of June
